plugins/reports/api/api.js

```diff
@@ -337,6 +380,15 @@ var logpath = path.resolve(__dirname, '../../../log/countly-api.log');
             return true;
 
     }
+
+    function validateUserForEmails(params){
+        if (!params.member.global_admin){
+            common.returnMessage(params, 401, 'User does not have right to access this information');
+            return false;
+        }else{
+            return true;
+        }
+    }
```

Comment:
Can you explain this validation? So only users with global admin access are able to create none core email reports?